### PR TITLE
added return statement in send methods

### PR DESCRIPTION
**Pull request by grptx**
the send and sendMultiple methods of Mailer class should be return a value

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
   "require": {
     "yiisoft/yii2": "~2",
     "bazilio/yii2-async": "^0.2.0",
-    "yarcode/yii2-daemon": "^0.3"
+    "yarcode/yii2-daemon": "^0.3",
+    "yarcode/yii2-async-mailer": "*"
```
Comment (metalagman) [inline composer.json]: self reference?

File: composer.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "yarcode/yii2-async-mailer",
+  "name": "grptx/yii2-async-mailer",
```
Comment (metalagman) [inline composer.json]: wrong package name

**Review by metalagman — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (grptx) [reply composer.json]: sorry it's my first pull request 
the change I wanted to propose it's only the first one: added return statement in send methods

**Comment by grptx** [comment]
I fixed the composer.json file pls consider to merge

**Pull request merged**